### PR TITLE
deps(webrtc): update webrtc to 0.8.0

### PR DESCRIPTION
**Pull request by melekes**
## Change checklist

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] A changelog entry has been made in the appropriate crates

### CODE REVIEW

File: transports/webrtc/CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,11 @@
 
 - Raise MSRV to 1.65.
   See [PR 3715].
+- Upgrade `webrtc` to `0.8.0`.
+  See [PR 4099].
```
Comment (thomaseizinger) [inline transports/webrtc/CHANGELOG.md]: It is completely encapsulated so no need for a changelog entry.

**Review by thomaseizinger — APPROVED** [review]
Thanks! Finally an update that actually works!

**Comment by mxinden** [comment]
Wonderful. Thank you @melekes!

**Comment by bkchr** [comment]
@thomaseizinger @mxinden will there be some minor release that includes this PR?

**Comment by mxinden** [comment]
@bkchr done with https://github.com/libp2p/rust-libp2p/pull/4123.

**Comment by bkchr** [comment]
Nice! Ty :)

**Pull request merged**